bundles/org.openhab.binding.openthermgateway/README.md

```diff
@@ -1,195 +1,432 @@
+        Switch item="CentralHeating2Enabled" icon="switch" label="Central Heating 2 Enabled"
+        Switch item="CentralHeating2EnabledThermostat" icon="switch" label="Central Heating 2 Enabled Thermostat"
+        Switch item="CentralHeating2Overridden" icon="switch" label="Central Heating 2 Overridden"
+        Switch item="CentralHeatingActive" icon="switch" label="Central Heating Active"
+        Switch item="DomesticHotWaterEnabled" icon="switch" label="Domestic Hot Water Enabled"
+        Switch item="DomesticHotWaterActive" icon="switch" label="Domestic Hot Water Active"
+        Switch item="BurnerActive" icon="switch" label="Burner Active"
+        Text item="RelativeModulationLevel" label="Relative Modulation Level [%.1f %%]"
+        Text item="MaximumRelativeModulationLevel" label="Maximum Relative Modulation Level [%.1f %%]"
+        Switch item="FaultIndication" icon="switch" label="Fault Indication"
+        Switch item="ServiceRequired" icon="switch" label="Service Required"
+        Switch item="Lockout-ResetEnabled" icon="switch" label="Lockout-Reset Enabled"
+        Switch item="LowWaterPressureFault" icon="switch" label="Low Water Pressure Fault"
+        Switch item="GasOrFlameFault" icon="switch" label="Gas Or Flame Fault"
+        Switch item="AirPressureFault" icon="switch" label="Air Pressure Fault"
+        Switch item="WaterOver-TemperatureFault" icon="switch" label="Water Over-Temperature Fault"
+        Text item="OEMFaultCode" label="OEM Fault Code"
+        Text item="UnsuccessfulBurnerStarts" label="Unsuccessful Burner Starts"
+        Text item="BurnerStarts" label="Burner Starts"
+        Text item="CentralHeatingPumpStarts" label="Central Heating Pump Starts"
+        Text item="DomesticHotWaterPump/ValveStarts" label="Domestic Hot Water Pump/Valve Starts"
+        Text item="DomesticHotWaterBurnerStarts" label="Domestic Hot Water Burner Starts"
+        Text item="BurnerHours" label="Burner Hours"
+        Text item="CentralHeatingPumpHours" label="Central Heating Pump Hours"
+        Text item="DomesticHotWaterPump/ValveHours" label="Domestic Hot Water Pump/Valve Hours"
+        Text item="DomesticHotWaterBurnerHours" label="Domestic Hot Water Burner Hours"
+        Text item="TransparentSlaveParameterNumber" label="Transparent Slave Parameter Number"
+        Text item="TransparentSlaveParameterEntry" label="Transparent Slave Parameter Entry"
+        Text item="FaultHistoryBufferNumber" label="Fault History Buffer Number"
+        Text item="FaultHistoryBufferEntry" label="Fault History Buffer Entry"
     }
 }
 ```
+
+### Ventilation / Heat Recovery
+
+#### Configuration
+
+The `ventilationheatrecovery` thing does not require any configuration settings.
+
+#### Channels
+
+The `ventilationheatrecovery` thing supports the following channels:
+
+| Channel ID | Item Type | Description | Access |
+|------------|-----------|-------------|--------|
+| vh_ventilationenable | Switch | Ventilation enabled | R |
```

Comment:
What's the reason for adding a suffix to the channel names?

Reply:
The names of the channels are mostly a copy/paste from the OpenTherm specifications. And those names do not always follow openHAB coding guidlines when it comes to the number of words, length in characters or things like using a suffix. So I find myself abbreviating or leaving words out.. just to comply with coding guidlines really but not making anything better in terms of readability imho or matching with the OT specs. So.. I don't know.. should I change them?

Reply:
I'd remove the prefix as it seems redundant for all Channels.